community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
+Generally, registration can be reimbursed if the attendee is speaking on behalf of OpenJS, the CPC, or OpenJS-hosted projects.
+Funding is not transferable to another person, or any other event. 
+
+Travel fund assistance can only be received once per year.
+
+## How To Apply
+
+### Submit Travel Fund Request
+
+Submit your travel fund request to the Cross Project Council by completing [this application](https://forms.gle/QDt3iqoXXB5Ycovz8).
+Please see [Deadlines and Timelines](#deadlines-and-timelines) for information about cut-off dates. 
+
+### Wait For Approval
+
+The Cross Project Council reviews requests once per month. If your request is approved, you will be notified via email.
+
+## Getting Reimbursed
+
+### Submitting Expenses
+
+After the event, kindly submit your expenses within 30 days:
+
+* Fill out these [two forms](https://drive.google.com/drive/folders/1E-dTuqnIWpZN2NP-1ioAWzK_W2zSwa_P?usp=share_link)
```

Comment:
There are three docs in that folder. Could we maybe link directly to the two forms?

Reply:
One of those are directions. I would prefer to just link to this directory.

Reply:
Can we maybe have a perma-link (shortened link) to this doc? Is this document under someone's individual account or some Foundation-shared folder?

Reply:
I feel that we might actually want to upload these docs in this repository just to ensure they're always available. And yes, it'd be great if we were more explicit about what each form is.

Reply:
Good call I will sort in a subsequent PR.